src/gluonts/dataset/split/splitter.py

```diff
@@ -250,13 +249,24 @@ def rolling_split(
                 )
                 prediction_length = getattr(self, "prediction_length")
 
-                assert (
-                    train.end + train.end.freq * prediction_length <= test.end
+                self._check_split_length(
+                    train.end, test.end, prediction_length
                 )
                 split._add_test_slice(test)
 
         return split
 
+    @staticmethod
+    def _check_split_length(
+        train_end: pd.Period, test_end: pd.Period, prediction_length: int
+    ) -> None:
+        msg = (
+            "Not enough observations after the split point to construct"
+            + " the test instance; consider using longer time series,"
+            + " or splitting at an earlier point."
```

Comment:
```suggestion
            "Not enough observations after the split point to construct"
            " the test instance; consider using longer time series,"
            " or splitting at an earlier point."
```

You can leave the plusses out. Some people don't like it, but I do :)

Reply:
Yes, an ignore options would be nice. I ll do that towards the end of the week/next week. I also want to make `item_id` optional. Currently it's mandatory.